### DISCUSSION TITLE
DysplayCAL and G-helper

### DISCUSSION THREAD

**Discussion by LoreAPC** [seerge/g-helper#2300] [Q&A]
Hi all! I have color profiled and calibrated my monitor with DisplayCAL and I use their profile loader to manage the profile associations.

Will the recently added G-helper features (profiles switching) interfere with this? Can I deactivate the feature?

Thank you very much!

**Comment by seerge** [comment]
@LoreAPC just set / keep Visual Mode on ``Default``, then app would not set any modes on start.

If you were using a custom color profiles before (i.e. not ones from Asus), just select and activate them again under windows ``Color Management`` application.
